### PR TITLE
Exclude JDK 8 - temurin, adopt-openj9 on macos, use defaults values

### PR DESCRIPTION
**Pull request by slawekjaranowski**

### CODE REVIEW

**Review by elharo — APPROVED** [review]

File: .github/workflows/maven-verify.yml [hunk 1]
```diff
@@ -28,7 +28,9 @@ jobs:
     with:
       # maven-args: "-Dinvoker.parallelThreads=2" cannot do this as this generate some concurrent download issues
       jdk-distribution-matrix: '[ "temurin", "zulu", "microsoft", "adopt-openj9" ]'
-      ff-jdk: "20"
-      jdk-matrix: '[ "8", "11", "17", "20" ]'
-      matrix-exclude: '[{ "jdk": "8", "distribution": "microsoft"},{ "jdk": "20", "distribution": "microsoft"},{ "jdk": "20", "distribution": "adopt-openj9"},]'
-      maven-matrix: '[ "3.6.3", "3.8.8", "3.9.5" ]'  # Maven versions matrix for verify builds
+      matrix-exclude: '[
```
Comment (olamy) [inline .github/workflows/maven-verify.yml:31]: the goal was to have a maximum of jdk to be tested as compiling can be highly dependant of the used jdk.
so with this we are really reducing the coverage.
could we keep testing on 11 maybe?

File: .github/workflows/maven-verify.yml [hunk 2]
```diff
@@ -28,7 +28,9 @@ jobs:
...
+          { "jdk": "8", "distribution": "microsoft" },
```
Comment (elharo) [inline .github/workflows/maven-verify.yml:32]: Why is this one excluded? It's not a Mac

File: .github/workflows/maven-verify.yml (hunk 1 above)
Comment (slawekjaranowski) [reply .github/workflows/maven-verify.yml:32]: done

File: .github/workflows/maven-verify.yml (hunk 2 above)
Comment (slawekjaranowski) [reply .github/workflows/maven-verify.yml:33]: It has been excluded early ... probably microsoft distribution does not support jdk 8 at all

File: .github/workflows/maven-verify.yml (hunk 1 above)
Comment (olamy) [reply .github/workflows/maven-verify.yml:32]: @slawekjaranowski Thanks! I was thinking of m-javadoc-p if we have similar "cleanup" :)

**Pull request merged**